src/flownet/config_parser/_config_parser.py

```diff
@@ -376,7 +382,18 @@ def _to_abs_path(path: Optional[str]) -> str:
                         MK.Description: "Number of additional nodes to create "
                         "(using Mitchell's best candidate algorithm)",
                     },
-                    "hull_factor": {MK.Type: types.Number, MK.Default: 1.2},
+                    "place_nodes_in_volume_reservoir": {
+                        MK.Type: types.Bool,
+                        MK.AllowNone: True,
+                        MK.Description: "When true use boundary of reservoir/layer volume as "
+                        "bounding cube to place initial candidates instead of concave hull of well perforations",
```

Comment:
The individual cells are "cube-like", but the sum of the cubes not. So maybe this fits better?

```suggestion
                        "bounding volume to place initial candidates instead of concave hull of well perforations.",
```